learning_resources/migrations/0085_add_content_summarizer_configs.py

```diff
@@ -0,0 +1,115 @@
+                                ("video", "video"),
+                                ("pdf", "pdf"),
+                            ],
+                            max_length=128,
+                        ),
+                        blank=True,
+                        default=list,
+                        null=True,
+                        size=None,
+                    ),
+                ),
+                (
+                    "allowed_extensions",
+                    django.contrib.postgres.fields.ArrayField(
+                        base_field=models.CharField(
+                            choices=[
+                                (".csv", ".csv"),
+                                (".doc", ".doc"),
+                                (".docx", ".docx"),
+                                (".htm", ".htm"),
+                                (".html", ".html"),
+                                (".json", ".json"),
+                                (".m", ".m"),
+                                (".mat", ".mat"),
+                                (".md", ".md"),
+                                (".pdf", ".pdf"),
+                                (".ppt", ".ppt"),
+                                (".pptx", ".pptx"),
+                                (".ps", ".ps"),
+                                (".py", ".py"),
+                                (".r", ".r"),
+                                (".rtf", ".rtf"),
+                                (".sjson", ".sjson"),
+                                (".srt", ".srt"),
+                                (".txt", ".txt"),
+                                (".vtt", ".vtt"),
+                                (".xls", ".xls"),
+                                (".xlsx", ".xlsx"),
+                                (".xml", ".xml"),
+                                (".doc", ".doc"),
+                                (".docx", ".docx"),
+                                (".htm", ".htm"),
+                                (".html", ".html"),
+                                (".json", ".json"),
+                                (".md", ".md"),
+                                (".pdf", ".pdf"),
+                                (".ppt", ".ppt"),
+                                (".pptx", ".pptx"),
+                                (".rtf", ".rtf"),
+                                (".sjson", ".sjson"),
+                                (".srt", ".srt"),
+                                (".txt", ".txt"),
+                                (".vtt", ".vtt"),
+                                (".xml", ".xml"),
+                            ],
+                            max_length=128,
+                        ),
+                        blank=True,
+                        default=list,
+                        null=True,
+                        size=None,
+                    ),
+                ),
+                ("is_active", models.BooleanField(default=True)),
+                (
+                    "platform",
```

Comment:
In addition to platform, there's also "[Offered By](https://github.com/mitodl/mit-learn/blob/main/learning_resources/constants.py#L49-L63)" and "[ETL Source](https://github.com/mitodl/mit-learn/blob/main/learning_resources/etl/constants.py#L80-L92)" - the hq issue specifically mentions platform, but it might be worth double-checking with Ferdi about that.